README.md

```diff
@@ -1,83 +1,120 @@
+[![](https://img.shields.io/github/v/release/ScratchAddons/ScratchAddons?style=flat-square&logo=github&logoColor=white&label=GitHub&color=181717)](https://github.com/ScratchAddons/ScratchAddons/releases)
+
+[![](https://img.shields.io/github/license/ScratchAddons/ScratchAddons?style=flat-square)](https://github.com/ScratchAddons/ScratchAddons/blob/master/LICENSE)
 [![](https://img.shields.io/badge/chat-on_discord-7289da.svg?style=flat-square)](https://discord.gg/R5NBqwMjNc)
 [![](https://img.shields.io/badge/website-scratchaddons.com-ff7b26.svg?style=flat-square)](https://scratchaddons.com)
 
 ## About
 
-Scratch Addons is a WebExtension (supports both Chrome and Firefox). Scratch Addons' mission is to combine all existing Scratch extensions, userscripts and userstyles into a single easy-to-access place, while still letting users choose which ones to enable.
+Scratch Addons combines new and existing features and themes for the [Scratch](https://scratch.mit.edu) website and project editor into one easy-to-access and configurable browser extension. The mission is to provide a centralized, up-to-date platform for community development of new features and themes for Scratch.
+
+<div align="center">
+
+![Scratch Addons screenshot](.github/readme-images/screenshot.png)
+
+</div>
+
+### About addons
+
+An addon mainly consists of one or more [userscripts](https://scratchaddons.com/docs/develop/userscripts/) (written in JavaScript) or [userstyles](https://scratchaddons.com/docs/develop/userstyles/) (written in CSS) that run on the Scratch website or project editor.
+
+Each addon declares its own [addon manifest](https://scratchaddons.com/docs/reference/addon-manifest/) (`addon.json` file). This file specifies under which circumstances each one of its userscripts and userstyles should be injected into the page. It also contains user-facing information, such as the description of the feature, and more information about the addon's settings.
+
+Userscripts work similarly to [extension content scripts](https://developer.chrome.com/docs/extensions/content_scripts/), but they also have access to `addon.*` APIs. Userscripts can use these built-in utilities for various purposes: waiting until a certain element exists on the page, listening to URL change events, getting a reference to the Scratch VM object, etc.
+
+Addons are designed to be compatible with each other. They are also developed with performance, internationalization, accessibility, and privacy in mind.
+
+Read the [documentation](https://scratchaddons.com/docs/develop/getting-started/addon-basics/) for more information about addons.
+
+### About the extension
+
+The Scratch Addons browser extension provides a settings page where users can enable, disable and configure addon settings. The extension interprets addon manifests, stores the user's settings, and provides APIs to userscripts. A new version of the extension is sent to stores at least once a month.
```

Comment:
Might I suggest rewording the part where a new version is released at least once a month? I would also use the word "release" and mention how each major update releases with new features.

Reply:
Go ahead